pkg/cmd/server/origin/master_config.go

```diff
@@ -175,16 +177,18 @@ func BuildMasterConfig(options configapi.MasterConfig) (*MasterConfig, error) {
 	kubeletClientConfig := configapi.GetKubeletClientConfig(options)
 
 	// in-order list of plug-ins that should intercept admission decisions (origin only intercepts)
-	admissionControlPluginNames := []string{"ProjectRequestLimit", "OriginNamespaceLifecycle", "PodNodeConstraints", "BuildByStrategy", "OriginResourceQuota"}
+	admissionControlPluginNames := []string{"ProjectRequestLimit", "OriginNamespaceLifecycle", "PodNodeConstraints", "BuildByStrategy", imageadmission.PluginName, "OriginResourceQuota"}
 	if len(options.AdmissionConfig.PluginOrderOverride) > 0 {
 		admissionControlPluginNames = options.AdmissionConfig.PluginOrderOverride
 	}
 
+	registry := quota.NewRegistryForAdmission(privilegedLoopbackOpenShiftClient, options.ImagePolicyConfig.MaxImagesBulkImportedPerRepository)
```

Comment:
> Doesn't this mean that I'm setting a partially complete QuotaRegistry?

It's a complete registry for OpenShift resources.

> I don't see how this handles the kube types.

It doesn't. There's a second quota admission [plugin](https://github.com/openshift/origin/blob/master/Godeps/_workspace/src/k8s.io/kubernetes/plugin/pkg/admission/resourcequota/admission.go#L44) running for k8s resources.

> If it's too hard to build the whole thing, name this such that its obvious it only works for a subset of types.

Hmm, would `NewOriginQuotaRegistryForAdmission()` do?